app/src/main/res/layout/item_word_list_home.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
```

Comment:
WARNING: [UnusedResources] The resource `R.layout.item_word_list_home` appears to be unused
 Unused resources make applications larger and slow down builds.